src/components/MenuLink/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MenuLink as ReachMenuLink } from '@reach/menu-button';
+
+const noop = () => {};
+
+const MenuLink = ({ onSelect, children, isDisabled, href, as }) => (
+  <ReachMenuLink
+    className={isDisabled ? 'fdsMenuItem--disabled' : undefined}
+    onSelect={!isDisabled ? onSelect : noop}
+    aria-disabled={isDisabled}
+    href={href}
+    as={as}
+  >
+    {children}
+  </ReachMenuLink>
+);
+
+MenuLink.propTypes = {
+  /** selection callback */
+  onSelect: PropTypes.func.isRequired,
+
+  /** React children - any JSX can be used in a menu item */
+  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node])
+    .isRequired,
+
+  /** Styles menu item as disabled and disables selection events */
+  isDisabled: PropTypes.bool,
+
+  href: PropTypes.string,
```

Comment:
```suggestion
  /** When the `as` prop is set to use an `a` element, use this to set the `href` attribute */ 
  href: PropTypes.string,
  
  /** When the `as` prop is set to use a `Link` element, use this to set the `to` prop of the `Link` */
  to: PropTypes.string,
```

Reply:
This would technically be something like 

`/** When the `as` prop is set to use an `a` element or is undefined, use this to set the `href` attribute */`

And at that point I wonder if that makes it more confusing. I feel like if someone wanted to use MenuLink to render an anchor, if they see an href, they would assume adding the href outright works (which is true).